com.apifox.apifox.json

```diff
@@ -0,0 +1,133 @@
+                    "sha256": "32d03afb0b1109fa75a96f89dfc118015283a5343f2216ee4b3757c19992f75d"
+                }
+            ]
+        },
+        {
+            "name": "noto-font-cjk-bold",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/share/fonts",
+                "install -D NotoSansCJK-Bold.ttc /app/share/fonts/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://noto-website-2.storage.googleapis.com/pkgs/NotoSansCJK-Bold.ttc.zip",
+                    "sha256": "3cd895bce6765903b2eefcd90eea6ed372364e66d5b896ea3a594bd3e737eba0"
+                }
+            ]
+        },
+        {
+            "name": "noto-font-cjk-medium",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/share/fonts",
+                "install -D NotoSansCJK-Medium.ttc /app/share/fonts/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://noto-website-2.storage.googleapis.com/pkgs/NotoSansCJK-Medium.ttc.zip",
+                    "sha256": "d7590637168e1cff4fd3a15422e483308d7ba2a7120ccaf96ead274c3b40e1b3"
+                }
+            ]
+        },
+        {
+            "name": "apifox",
+            "builddir": true,
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 com.apifox.apifox.metainfo.xml -t $FLATPAK_DEST/share/metainfo",
+                "install -Dm644 com.apifox.apifox.desktop /app/share/applications/com.apifox.apifox.desktop",
+                "install -Dm644 com.apifox.apifox.svg /app/share/icons/hicolor/scalable/apps/com.apifox.apifox.svg",
+                "install -Dm755 run.sh /app/bin/run.sh",
+                "install -Dm755 apply_extra ${FLATPAK_DEST}/bin/apply_extra"
+
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "com.apifox.apifox.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "com.apifox.apifox.metainfo.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "com.apifox.apifox.svg"
+                },
+                {
+                    "type": "file",
+                    "path": "run.sh"
+                },
+                {
+                    "type": "extra-data",
+                    "filename": "apifox_linux.tar.xz",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://file-assets.apifox.com/download/Apifox-linux-manual-latest.tar.gz",
+                    "sha256": "792595282eab3fb2c2902d12b0495cc16cc3c65c763983eead49c0b6645beec4",
+                    "size": 272837328
```

Comment:
size changed, this is going to be a problem, can you set up rotating-url checker?

Reply:
https://github.com/flathub-infra/flatpak-external-data-checker?tab=readme-ov-file#url-checker

Also an [exception](https://docs.flathub.org/docs/for-app-authors/linter#exceptions) to https://docs.flathub.org/docs/for-app-authors/linter#flathub-json-automerge-enabled

Reply:
@jcleng you still need to add the exceptions PR and add automerge to flathub.json

Reply:
you still have to do the exceptions PR https://docs.flathub.org/docs/for-app-authors/linter/#exceptions